Formula/starship.rb

```diff
@@ -35,10 +35,26 @@ def install
 
     fish_output = Utils.safe_popen_read("#{bin}/starship", "completions", "fish")
     (fish_completion/"starship.fish").write fish_output
+
+    zsh_prompt = Utils.safe_popen_read("#{bin}/starship", "init", "zsh", "--print-full-init")
+    prompt_setup = "prompt_starship_setup"
+    (zsh_function/prompt_setup).write "#{prompt_setup}() {\n#{zsh_prompt}\n}\n#{prompt_setup}"
+  end
+
+  def caveats
+    <<~EOS
+      To activate Starship as a prompt theme, add the following to your .zshrc:
+        autoload -U promptinit
+        promptinit
+        prompt starship
+    EOS
```

Comment:
Is this specific to homebrew?

Reply:
@SMillerDev It's according to the [instructions here](http://zsh.sourceforge.net/Doc/Release/User-Contributions.html#Prompt-Themes).

Reply:
Then it isn't an extrordinary circomstance caused by the homebrew install (and therefore not a caveat)

Reply:
@SMillerDev I see… Resolved.

Reply:
I suspect it might be Homebrew-specific actually. At the very least, it strikes me as very much like installing completions, which Homebrew does display caveats for.